doc/migrate-to-pymongo4.rst

```diff
@@ -420,6 +420,17 @@ instead.
 
 .. _group command: https://docs.mongodb.com/manual/reference/command/group/
 
+Collection.map_reduce and Collection.inline_map_reduce are removed
+..................................................................
+
+Removed :meth:`pymongo.collection.Collection.map_reduce` and
+:meth:`pymongo.collection.Collection.inline_map_reduce`.
+Migrate to :meth:`~pymongo.collection.Collection.aggregate` or run the
```

Comment:
This should link to the database documentation for the map/reduce agg pipeline stages.
https://docs.mongodb.com/manual/core/aggregation-pipeline/#aggregation-pipeline-as-an-alternative-to-map-reduce
https://docs.mongodb.com/manual/reference/aggregation-commands-comparison/

Reply:
Done.